modyn/trainer_server/internal/dataset/online_dataset.py

```diff
@@ -305,6 +312,11 @@ def callback_func() -> None:
 
                 callback = callback_func
 
+            # We implement shuffling on a partition level by mapping everything to increasing indices but actually load
+            # different partition data.
+            shuffle_partition = (
+                self._shuffled_partition_indices[self._next_partition_to_fetch] if self._shuffle else None
+            )
```

Comment:
Nit: this confuses me for a bit. better to rename it `shuffle_partition_id` as that's the name in `_get_data`

Reply:
done